main.go

```diff
@@ -185,6 +257,21 @@ func main() {
 			if installPage != "" {
 				publicInstallPages[filepath.Base(pth)] = installPage
 			}
+		case ".zip":
```

Comment:
please make sure to follow the original task description, in this case other systems are relying on this schemantic: `*.xcarchive.zip`, so you can check for the extension `.xcarchive.zip` and then don't do `xcarchive.CheckForXcarchive` because this extension is held like this to be able to identify xcarchives easily

Reply:
Updated code.